src/helpers/FontUtils.hpp

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#include <string>
+
+int count_missing_glyphs(const std::string& text, const std::string& fontFamily);
```

Comment:
I'd avoid global funcs tbh. Why not `namespace FontUtils`?

Reply:
Let me say, should we just write an inline function, and reuse the cairo context, pango layout and font description in `drawNotifications()`, as it's called very frequent, the `iconBackendFromFont()` and `countMissingGlyphs()` are not likely to be reused anywhere else.